src/grisp_tools_util.erl

```diff
@@ -88,12 +88,18 @@ cdn_path(otp, #{platform := Platform} = State) ->
     Path = lists:join($/, ["platforms", atom_to_binary(Platform), "otp", File]),
     uri_string:resolve(Path, [env(cdn), "/"]).
 
-paths(Root, Platform, Version) ->
+paths(Root, Platform, Version, _Hash, _CustomBuild = true) ->
     Dir = otp_dir(Root, Platform, Version),
+    sub_paths(Dir, "");
```

Comment:
Maybe cleaner to do `sub_paths([Dir])` and `sub_paths([Dir, Hash])` and later:
```erl
sub_paths(Dirs) ->
    #{
        build => filename:join(Dirs ++ ["build"]),
        ...
```

Reply:
cool idea